src/Features/CSharp/Portable/Formatting/CSharpAccessibilityModifiersNewDocumentFormattingProvider.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Composition;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.AddAccessibilityModifiers;
+using Microsoft.CodeAnalysis.CodeStyle;
+using Microsoft.CodeAnalysis.Editing;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.LanguageServices;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+
+namespace Microsoft.CodeAnalysis.Formatting
+{
+    [Export(LanguageNames.CSharp, typeof(INewDocumentFormattingProvider)), Shared]
```

Comment:
this seems weird.  i would expect us to have a specific export for docformattingproviders.  similar to ExportCodeRefactoringProvider.

Reply:
Thank you, I'm very happy that I wasn't too far off, as this was one of the things I considered. It didn't seem like this feature was "big enough" for its own Export attribute, but happy to add it. Definitely makes things easier.